### PR TITLE
migrate some recipes to RA2

### PR DESCRIPTION
**Pull request by boubou19**
- Migrate recipes from AssemblerRecipes.java
- Migrate recipes from MixerRecipes.java
- Migrate recipes from FluidSolidifierRecipes.java
- Migrate recipes from FormingPressRecipes.java
- Migrate recipes from FluidExtractorRecipes.java
- Migrate recipes from ImplosionCompressorRecipes.java
- Materials.Superconductor -> Materials.SuperconductorUHV

### CODE REVIEW

File: src/main/java/com/dreammaster/bartworksHandler/BioItemLoader.java [hunk 1]
```diff
@@ -99,18 +102,9 @@ public class BioItemLoader {
...
-                NI,
-                NI,
-                NI,
-                NI,
-                GT_ModHandler.getDistilledWater(1000),
-                NF,
-                BIOTEMSSTACKS[2],
-                600,
-                480);
+        RA.stdBuilder().itemInputs(GT_Utility.getIntegratedCircuit(11), BIOTEMSSTACKS[1], NI, NI, NI, NI)
```
Comment (chochem) [inline src/main/java/com/dreammaster/bartworksHandler/BioItemLoader.java:105]: dont add nulls to iteminputs and fluidoutput

**Comment by chochem** [comment]
can you convert this one as well? https://github.com/GTNewHorizons/NewHorizonsCoreMod/blob/786cd78c89e0783889b545e909d6c4e8e395ac7d/src/main/java/com/dreammaster/gthandler/recipes/AssemblerRecipes.java#L1443

**Comment by chochem** [comment]
these should be TierEU.RecipeZPM: https://github.com/GTNewHorizons/NewHorizonsCoreMod/blob/786cd78c89e0783889b545e909d6c4e8e395ac7d/src/main/java/com/dreammaster/gthandler/recipes/AssemblerRecipes.java#L3240C63-L3240C63

5 occurences

**Comment by chochem** [comment]
and eut(500000) should be TierEU.RecipeUV
three times in the file.

File: src/main/java/com/dreammaster/gthandler/recipes/FluidSolidifierRecipes.java [hunk 2]
```diff
@@ -25,543 +28,468 @@ public class FluidSolidifierRecipes implements Runnable {
...
-                ItemList.Shape_Mold_Ball.get(0L),
-                new FluidStack(FluidRegistry.getFluid("dye.chemical.dyewhite"), 72),
-                GT_ModHandler.getModItem(GregTech.ID, "gt.metaitem.02", 1L, 32429),
-                100,
-                16);
+
+        GT_Values.RA.stdBuilder().itemInputs(ItemList.Shape_Mold_Ball.get(0L))
+                .itemOutputs(GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Sodium, 1L))
+                .fluidInputs(Materials.Sodium.getFluid(1000)).noFluidOutputs().duration(20 * TICKS).eut(4)
+                .addTo(sFluidExtractionRecipes);
```
Comment (chochem) [inline src/main/java/com/dreammaster/gthandler/recipes/FluidSolidifierRecipes.java:35]: this one is more major: all of these need to be sFluidSolidficationRecipes.

Comment (boubou19) [reply src/main/java/com/dreammaster/gthandler/recipes/FluidSolidifierRecipes.java:35]: whoopsie, will fix it in a second

**Review by chochem — CHANGES_REQUESTED** [review]
currently crashes in full game

**Comment by chochem** [comment]
most likely caused by these 4 recipes https://github.com/GTNewHorizons/NewHorizonsCoreMod/blob/786cd78c89e0783889b545e909d6c4e8e395ac7d/src/main/java/com/dreammaster/gthandler/recipes/AssemblerRecipes.java#L745

File: src/main/java/com/dreammaster/gthandler/recipes/ImplosionCompressorRecipes.java [hunk 3]
```diff
@@ -23,318 +27,426 @@ public class ImplosionCompressorRecipes implements Runnable {
...
-                    CustomItemList.DeshDualCompressedPlates.get(1L),
-                    GT_OreDictUnificator.get(OrePrefixes.dustTiny, Materials.Ash, 2L));
+
+            GT_Values.RA.stdBuilder()
+                    .itemInputs(GT_ModHandler.getModItem(GalacticraftMars.ID, "item.itemBasicAsteroids", 2L, 6))
+                    .itemOutputs(
+                            CustomItemList.TitaniumDualCompressedPlates.get(1L),
+                            GT_OreDictUnificator.get(OrePrefixes.dustTiny, Materials.Ash, 2L))
+                    .noFluidInputs().noFluidOutputs().duration(20 * TICKS).eut(TierEU.RECIPE_LV)
+                    .metadata(ADDITIVE_AMOUNT, 2).addTo(sImplosionRecipes);
```
Comment (chochem) [inline src/main/java/com/dreammaster/gthandler/recipes/ImplosionCompressorRecipes.java:156]: none of the recipes with amount 2 appear ingame

File: src/main/java/com/dreammaster/gthandler/recipes/ImplosionCompressorRecipes.java [hunk 4]
```diff
@@ -23,318 +27,426 @@ public class ImplosionCompressorRecipes implements Runnable {
...
-                56,
-                CustomItemList.HeavyDutyPlateTier7.get(1L),
-                GT_OreDictUnificator.get(OrePrefixes.dustTiny, Materials.Neutronium, 7L));
-        GT_Values.RA.addImplosionRecipe(
-                CustomItemList.HeavyDutyAlloyIngotT8.get(1L),
-                64,
-                CustomItemList.HeavyDutyPlateTier8.get(1L),
-                GT_OreDictUnificator.get(OrePrefixes.dustTiny, Materials.BlackPlutonium, 8L));
+
+        GT_Values.RA.stdBuilder().itemInputs(CustomItemList.HeavyDutyAlloyIngotT5.get(1L))
```
Comment (chochem) [inline src/main/java/com/dreammaster/gthandler/recipes/ImplosionCompressorRecipes.java:275]: tier 4 is fine for some reason, but tier5-8 do not appear ingame

Comment (boubou19) [reply src/main/java/com/dreammaster/gthandler/recipes/ImplosionCompressorRecipes.java:275]: tier 4 is handled in BW it seems, at least not in the ImplosionCompressorRecipes.java

**Pull request merged**